examples/scripts/sft_hpu_example.py

```
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)